hazelcast/src/main/resources/hazelcast-client-config-4.0.xsd

```diff
@@ -249,14 +249,19 @@
     </xs:complexType>
 
     <xs:complexType name="load-balancer">
-        <xs:attribute name="type" use="required">
-            <xs:simpleType>
-                <xs:restriction base="non-space-string">
-                    <xs:enumeration value="random"/>
-                    <xs:enumeration value="round-robin"/>
-                </xs:restriction>
-            </xs:simpleType>
-        </xs:attribute>
+        <xs:simpleContent>
```

Comment:
Same question as above about changing the released version's `.xsd` file

Reply:
Rolled back the change as suggested.